### DISCUSSION TITLE
I am getting type errors when trying to use arguments. Not sure what I am doing wrong.

### DISCUSSION THREAD

**Discussion by 0b3r** [ivanhofer/typesafe-i18n#664] [Q&A] [answered]
Given the following English base translation:

![image](https://user-images.githubusercontent.com/8107906/233513173-104bf455-b476-4dce-bf1a-13b21e949a8e.png)

My Canadian French and Spanish translations are showing the same type errors:

![image](https://user-images.githubusercontent.com/8107906/233513269-08a321de-0676-4355-b066-3df4f8d4b1d8.png)
![image](https://user-images.githubusercontent.com/8107906/233513286-bceae16b-88d9-4784-aa71-dffd09682ba9.png)

Here are the `i18n-types.ts` that are generated:
![image](https://user-images.githubusercontent.com/8107906/233513375-94ebfe09-494a-4de6-9d3e-0faa1d1785c8.png)

I am not sure what I am doing wrong? Has anyone else come across this issue?

More info:

`"typescript": "^5.0.3"`
`"typesafe-i18n": "^5.24.3"`

**Comment by ivanhofer** [comment, marked as answer]
Hi, you only have to specify the types in your `BaseTranslation`. Just remove `:string` from french and spanish and it should not show an error anymore :)

**Reply by 0b3r** [reply]
Amazing! Thank you so much!!!
